pr-db-mongo/requirements.txt

```diff
@@ -0,0 +1 @@
+pymongo==4.4.1
```

Comment:
```suggestion
```
We no longer use `pymongo` I believe.

Reply:
Needed when dropping database

Reply:
Since we **have** to use mongo tools already for DB cloning, can't we also use [them to drop the database](https://stackoverflow.com/a/8857373)?